lib/rubocop/cop/offense_message_annotater.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # A annotater that takes the basic message from a cop
+    # and annotates it for output based on options or settings.
```

Comment:
I added documentation to public methods and used yard doc syntax. If that is not as much addition to the class documentation, please let me know what other information should be included.